torch/_inductor/scheduler.py

```diff
@@ -663,7 +663,7 @@ def set_last_usage(
         super().set_last_usage(future_used_buffers, mutation_real_name)
         # Set self.last_usage on the snodes
         # This will be used for optimisations within the kernel
-        future_used_buffers = set()
+        future_used_buffers = set() # type: ignore[var-annotated]
```

Comment:
This just needs to be specified as what type of set it is. Ie Set[Buffer] or whatever the proper type is.